app/src/main/res/values/strings.xml

```diff
@@ -1897,8 +1897,8 @@
 	<!-- Dialog button text for canceling the rename top site prompt. -->
 	<string name="top_sites_rename_dialog_cancel">Cancel</string>
 
-    <!-- In-activate tabs in the tabs tray -->
-    <!-- Title text displayed in the tabs tray when a tab has been unused for 4 days. -->
+    <!-- Inactive tabs in the tabs tray -->
+    <!-- Title text displayed in the tabs tray when a tab has been unused for 14 days. -->
```

Comment:
Was this string added recently (last sprint)? If it's older we want to deprecate the old one and add this new one so that "14 days" can be translated correctly. Let me find the wiki..

Reply:
https://github.com/mozilla-mobile/fenix/wiki/Remove-strings

Reply:
I don't think we need to have these re-translated because the context is still the same as before.

Reply:
I wasn't sure if changing the description would make it repopulate the string in l10n's system